kinesis-video-gst-demo/kinesis_video_gstreamer_sample_multistream_app.cpp

```diff
@@ -176,6 +176,9 @@ typedef struct _CustomData {
     map<string, bool> stream_started;
     map<string, uint8_t*> frame_data_map;
     map<string, UINT32> frame_data_size_map;
+    // Pts of first frame
+    map<string, uint64_t> first_pts_map;
+    map<string, uint64_t> producer_start_time_map;
```

Comment:
This sample is meant for streaming multiple RTSP streams at the same time. In that case I think the start time should be the same for all streams.

Reply:
Shouldn't it be unique for each RTSP stream? The frames coming from different RTSP url might not start flowing at the same start time, the compensation will force all the RTSP stream starting from first one's start time if we only keep one producer_start_time.

Reply:
make sense